Index: BUILD/Makefile.inc
===================================================================
--- BUILD.orig/Makefile.inc
+++ BUILD/Makefile.inc
@@ -38,6 +38,7 @@ CFLAGS += -I. -I../include -fno-strict-a
 	-Wall -Wstrict-prototypes -Werror -Wformat-security \
 	-D_GNU_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 \
 	-DPRODUCT_NAME_SHORT=\"$(PRODUCT_NAME_SHORT)\"
+CFLAGS += -I/usr/include/libxml2
 CFLAGS += $(RPM_OPT_FLAGS) $(CPPFLAGS)
 CFLAGS += $(if $(PLOOP_LOG_FILE), -DPLOOP_LOG_FILE=\"$(PLOOP_LOG_FILE)\")
 #CFLAGS += -DDEBUG_TIMES
